sample-libs/Samples.DatabaseHelper/Extensions.cs

```diff
@@ -1,30 +1,29 @@
 using System.Collections.Generic;
 using System.Data;
-using System.Data.Common;
 
 namespace Samples.DatabaseHelper
 {
     public static class Extensions
     {
-        public static IEnumerable<IDataRecord> AsDataRecords(this DbDataReader reader)
+        public static IEnumerable<IDataRecord> AsDataRecords(this IDataReader reader)
```

Comment:
Given the potential issues we have seen with multiple versions (HttpClient)
Should we default to not using static refs?

Reply:
Not a blocker for this PR IMO. Just starting the discussion.

Reply:
I created a ticket to follow up on this.